postgres-9.5.json

```diff
@@ -0,0 +1,41 @@
+{
+  "PostgreSQL": {
+    "description": "PostgreSQL (9.5.21), open-source relational DBMS.<p>Based on the official PostgreSQL docker image: <a href='https://hub.docker.com/_/postgres' target='_blank'>https://hub.docker.com/_/postgres</a>.</p><p><strong>Note: </strong>Note: This rock-on requires Rockstor version 3.9.2-53 or later.</p>",
+    "version":"1.2",
+    "website":"https://www.postgresql.org",
+    "more_info": "<h4>Important locations inside docker image:</h4><p>Config. file:<code>/var/lib/pgsql/data/pg_hba.conf</code></p> <p>Databases: <code>/var/lib/pgsql/data</code></p> <p>Logs: <code>/var/log/postgresql</code></p>",
+    "volume_add_support":true,
+    "containers": {
+        "postgresql": {
+        "image": "postgres",
+        "launch_order":1,
+        "tag": "9.5.21",
+        "ports": {
+            "5432": {
+                "description":"PostgreSQL port. Suggested default: 5433 (Rockstor typically uses 5432 already)",
+                "host_default":5433,
+                "label":"PostgreSQL port",
+                "protocol":"tcp"
+            }
+        },
+        "volumes": {
+            "/var/lib/postgresql/data":{
+                "description":"Choose a share where the database should be stored. E.g.: create a share called postgresql-share1 for this purpose alone.",
```

Comment:
That makes sense. I had originally kept it generic, because I had hoped we can get to a version-independent Postgres rockon. But, as I found out the tag is really they driver and that's not (yet) exposed, so I'm fine with this change.